refresh.py

```diff
@@ -538,13 +553,14 @@ def get_mteb_average(task_dict: dict) -> tuple[Any, dict]:
     DATA_OVERALL.insert(
         1,
         f"Average ({len(all_tasks)} datasets)",
-        DATA_OVERALL[all_tasks].mean(axis=1, skipna=False),
+        DATA_OVERALL[find_tasks(DATA_OVERALL.columns, all_tasks)].mean(axis=1, skipna=False),
```

Comment:
was there something wrong beforehand?

Reply:
In this line, all_tasks have task from config and for each task appended. For example, `GeoreviewClassification` in `all_tasks` is `GeoreviewClassification (rus-Cyrl)` in `DATA_OVERALL`. Previously only french datasets had lang in their tasks, but they presented in `config.yaml`. I can change dataset names in config